docker-readme/auto-setup.md

```diff
@@ -0,0 +1,143 @@
+Temporal requires an RDBMS backend; this Docker setup script supports Cassandra (default), MySQL, or PostgreSQL.
+
+Optionally also supports Elasticsearch for [visibility](https://docs.temporal.io/self-hosted-guide/visibility).
+
+## Environment Variables
+
+`DB` (default: `cassandra`)
+
+This variable specifies the type of database you're connecting to. Allowed values are `cassandra`, `mysql`, and `postgres`.
+
+`SKIP_SCHEMA_SETUP` (default: `false`)
+
+This variable specifies whether database schema creation should be skipped. Set this to true if you're using an existing schema.
+
+`SKIP_DB_CREATE` (default: `false`)
+
+This variable specifies whether database creation should be skipped. Set this to true if you're using an existing database.
+
+### Cassandra
+
+These options only apply to the Cassandra database backend.
+
+`KEYSPACE` (default: `temporal`)
+
+This variable specifies the name of your Cassandra keyspace.
+
+`CASSANDRA_SEEDS` (defaults to unset)
+
+`CASSANDRA_PORT` (default: 9042)
+
+This variable specifies the port to connect to Cassandra on.
+
+`CASSANDRA_USER` (defaults to unset)
+
+This variable specifies your Cassandra username.
+
+`CASSANDRA_PASSWORD` (defaults to unset)
+
+This variable specifies your Cassandra password.
+
+`CASSANDRA_TLS_ENABLED` (defaults to unset)
```

Comment:
would probably say "false" here for clarity although you are technically correct.